src/main/resources/callCachingWorkflows/dockerHashNoLookup.test

```diff
@@ -0,0 +1,13 @@
+name: dockerHashNoLookup
+backends: [LocalNoDocker]
+
+files {
+  wdl: dockerHashNoLookup/dockerHashNoLookup.wdl
+  options: dockerHashNoLookup/dockerHashNoLookup.options
+}
+
+metadata {
+  workflowName: docker_hash_no_lookup
+  status: Succeeded
+  "calls.docker_hash_no_lookup.dockerhub.callCaching.result": "Cache Hit: <<CACHE_HIT_UUID>>:docker_hash_no_lookup.dockerhub:-1"
```

Comment:
This is not a normal test it's a Call Caching test. This runs it twice https://github.com/broadinstitute/centaur/pull/179/files#diff-bb3a325ecde8996697efc6b9307f49baR35

I guess the naming of the test is misleading, it's supposed to check both that the docker tag is not looked up and then despite that call caching still works.